pkg/core/native/management.go

```diff
@@ -719,12 +717,10 @@ func (m *Management) emitNotification(ic *interop.Context, name string, hash uti
 
 func checkScriptAndMethods(script []byte, methods []manifest.Method) error {
 	l := len(script)
-	offsets := bitfield.New(l)
 	for i := range methods {
 		if methods[i].Offset >= l {
 			return fmt.Errorf("method %s/%d: offset is out of the script range", methods[i].Name, len(methods[i].Parameters))
 		}
-		offsets.Set(methods[i].Offset)
 	}
-	return vm.IsScriptCorrect(script, offsets)
+	return nil
```

Comment:
Well, it's bad. The only reason we have it is delayed 3.6.

Reply:
Discussed, we still need our T5 testnet node to be alive until 3.6.0 release and by the time SC coding workshop starts.